src/main/kotlin/com/openlattice/entitysets/controllers/EntitySetsController.kt

```diff
@@ -519,11 +519,25 @@ constructor(
             @RequestBody dateTime: String
     ): Set<UUID> {
         ensureReadAccess(AclKey(entitySetId))
-        val es = getEntitySet(entitySetId)
+        val es = entitySetManager.getEntitySet(entitySetId)!!
         check(es.hasExpirationPolicy()) { "Entity set ${es.name} does not have an expiration policy" }
 
         val expirationPolicy = es.expiration
         val expirationPT = expirationPolicy.startDateProperty.map { edmManager.getPropertyType(it) }
+
+        recordEvent(
+                AuditableEvent(
+                        spm.currentUserId,
+                        AclKey(entitySetId),
+                        AuditEventType.READ_ENTITY_SET,
+                        "EntitySetApi.getExpiringEntitiesFromEntitySet() resulted in an EntitySet READ",
```

Comment:
Not 100% sure about the audit type and message, but otherwise lgtm

Reply:
(also super minor and I realize these are all wrong in the controller but it should be EntitySetsApi instead of EntitySetApi)

Reply:
Id have to create a new AuditEventType if the current one isn't right (which I agree it's a little off). Does it warrant its own type? I can change the message to something like "returned entity key ids" or "returned expiring entity key ids"?

Reply:
@katherinebernstein ?

Reply:
I'm not sure how we need to audit this

Reply:
oh I just meant in the string on line 533

Reply:
but yeah I don't really see much harm in adding a new audit type -- I guess it would prevent filtering by all updates to entity set, but that's not exactly true because you can filter on all the more fine-grained audit types that you care about. so I'd vote for adding a new audit event type.

Reply:
> oh I just meant in the string on line 533

oh jk you did this change